ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+### Expected Behavior
+
+(Describe how things should work.)
+
+
+### Actual Behavior
+
+(Describe what went wrong.)
+
+
+### Steps to Reproduce the issue
+
+(Describe how someone can see a working example of the issue. Include what version of Ember.js/Ember CLI you are using, and any details regarding browser specific behaviors. Is there a feature flag that needs to be enabled to reproduce the issue?)
```

Comment:
I think one key will be for this to gently nudge the user to provide a better issue, without feeling imposing.